m0t1fs/linux_kernel/st/m0t1fs_client_inc.sh

```diff
@@ -799,32 +799,32 @@ m0t1fs_big_bs_io_test()
 m0t1fs_test_MOTR_2099()
 {
 	local rc=0
-	mount_m0t1fs $MOTR_M0T1FS_MOUNT_DIR || rc=1
+	mount_m0t1fs "$MOTR_M0T1FS_MOUNT_DIR" || rc=1
 
-	df $MOTR_M0T1FS_MOUNT_DIR
-	used_before=`df $MOTR_M0T1FS_MOUNT_DIR --output=used | tail -n 1`
+	df "$MOTR_M0T1FS_MOUNT_DIR"
+	used_before=`df "$MOTR_M0T1FS_MOUNT_DIR" --output=used | tail -n 1`
 	for i in 0:00{0..9}{0..1}; do
 		# This is to create 20 files, 200MB each, 8GB in total.
 		m0t1fs_file=$MOTR_M0T1FS_MOUNT_DIR/${i}
-		touch_file $m0t1fs_file 8192 && run "dd if=/dev/zero of=$m0t1fs_file bs=200M count=1" || rc=1
+		touch_file "$m0t1fs_file" 8192 && run "dd if=/dev/zero of=$m0t1fs_file bs=200M count=1" || rc=1
 	done
-	df $MOTR_M0T1FS_MOUNT_DIR
-	used_after=`df $MOTR_M0T1FS_MOUNT_DIR --output=used | tail -n 1`
+	df "$MOTR_M0T1FS_MOUNT_DIR"
+	used_after=`df "$MOTR_M0T1FS_MOUNT_DIR" --output=used | tail -n 1`
 	for i in 0:00{0..9}{0..1}; do
 		m0t1fs_file=$MOTR_M0T1FS_MOUNT_DIR/${i}
-		rm -f $m0t1fs_file
+		rm -f "$m0t1fs_file"
 	done
-	df $MOTR_M0T1FS_MOUNT_DIR
-	used_delete=`df $MOTR_M0T1FS_MOUNT_DIR --output=used | tail -n 1`
+	df "$MOTR_M0T1FS_MOUNT_DIR"
+	used_delete=`df "$MOTR_M0T1FS_MOUNT_DIR" --output=used | tail -n 1`
 	echo "used_before used_after used_delete $used_before $used_after $used_delete"
-	if [ $used_before -ne $used_delete ] ; then
+	if [ "$used_before" -ne "$used_delete" ] ; then
 		echo "balloc space leak? After deletion, used blocks are not the same as before."
 		rc=1
 	fi
 
-	total_blocks=$((200 * 20 * 1024)) #in 1K blocks
+	total_blocks=`expr 200 \* 20 \* 1024` #in 1K blocks
```

Comment:
replace "`" with $()

Why are you doing something in the opposite way?

Reply:
I did not change the $() with backticks, since the branch was stale it showed the now $() code in the left window. As there were too many failures when this file was changes I though it is because "*" was not escaped hence I had put a "\" before "*".  

Now leaving it unchanged.